planet/cli/data.py

```diff
@@ -309,13 +310,18 @@ async def search(ctx, item_types, filter, limit, name, sort, pretty):
     parameter will be applied to the stored quick search.
     """
     async with data_client(ctx) as cl:
-
+        item_ids = []
```

Comment:
```suggestion
In #645 we've recognized that search results may cover multiple item_types and that mixing their ids together sets a user up for failure. @kevinlacaille what would you think about this?

```suggestion
        if ids_only and len(item_types) > 1:
            raise ClickException("item id output is not allowed when multiple item types have been searched.")

        item_ids = []
```